src/nci_adapter.c

```diff
@@ -127,11 +138,151 @@ nci_adapter_intf_info_new(
         } else {
             info->activation_param.bytes = NULL;
         }
+
+        if(ntf->mode_param) {
```

Comment:
No need for this check, `nci_util_copy_mode_param` checks it and returns NULL.

Reply:
Indeed, thanks